tests/systemtests/rest_client.go

```
@@ -0,0 +1,158 @@
+	"strings"
+
+	"github.com/go-resty/resty/v2"
+)
+
+type restClient struct {
+	client            *resty.Client
+	networkIdentifier string
+}
+
+func newRestClient(rosetta rosettaRunner) *restClient {
+	client := resty.New().
```

golangci-lint (1.62.2)

SA1019: resty.New().
	SetHostURL is deprecated: use [Client.SetBaseURL] instead. To be removed in the v3.0.0 release. 

(staticcheck)